stacyp2006/IdeaBox
# upper-half-nav-input HTML

#### What's this PR do?

- Adds the basic elements of the static comp into the HTML (specifically for the elements in the upper half and nav
- Adds 'img' directory with Icon Images

#### Where should the reviewer start?

Start at index.html

#### How should this be manually tested?

Open index.html in browser. Tester should see:

- Hamburger Icon in nav
- Title text
- Title input field
- Body Text
- Body Input Field
- Save button
- Search Icon
- Search input field

#### Any background context you want to provide?

Nothing functional yet

#### What are the relevant tickets?

None at this time
